utils/generate_schema_yml.py

```diff
@@ -124,7 +124,9 @@ def generate_schema_dict(directory_path):
     if model_directory[:1] in ["0", "1"]:
         target = "cta"
     elif model_directory[:1] in ["2", "3"]:
-        target = "partner"
+        #target = "partner"
+        print("dbt tests currently do not work for partner models, skipping.")
+        return
```

Comment:
Should this be returning? It looks like its doing other stuff on top of this later in the script that might affect other tests??

Reply:
Oh, I was replicating the behavior it does for the CTE models (if you look a few lines up):

```
    if model_directory == "0_ctes":
        print("Codegen doesn't work on emphemeral models, skipping.")
        return
```

Reply:
oh nice i looked down but not up. that sounds like a life lesson lol